eng/testing/tests.wasm.targets

```diff
@@ -20,7 +20,7 @@
       - for AOT library tests, we use WasmNativeStrip=false, so we already have symbols
     -->
     <WasmNativeStrip Condition="'$(WasmNativeStrip)' == ''">false</WasmNativeStrip>
-    <WasmEmitSymbolMap Condition="'$(WasmEmitSymbolMap)' == '' and '$(RunAOTCompilation)' != 'true'">true</WasmEmitSymbolMap>
+    <WasmEmitSymbolMap Condition="'$(WasmEmitSymbolMap)' == ''">true</WasmEmitSymbolMap>
```

Comment:
Should this be reverted from the final PR ?

Reply:
I had added this because we don't get the symbols on aot now. This lets xharness symbolicate the traces. I think we can keep this.